### PR TITLE
Update flask.rst

### PR DESCRIPTION
**Pull request by ThinkDigitalSoftware**
Grammer update in docstring for hello()

### CODE REVIEW

File: docs/flask.rst [hunk 1]
```diff
@@ -154,7 +154,7 @@ For this we will update ``hello_flask.py``.
 
   @APP.route('/hello/<name>/')
   def hello(name):
-      """ Displays the page greats who ever comes to visit it.
+      """ Displays the page. Greets whoever comes to visit it.
```
Comment (maxking) [inline docs/flask.rst]: "Displays the page and greets whoever comes to visit"

I believe this would be a tad bit better.

**Comment by ThinkDigitalRepair** [comment]
Nope. I agree. Your way is even better

**Comment by kushaldas** [comment]
@ThinkDigitalRepair Can you please update the PR?

**Comment by ThinkDigitalRepair** [comment]
I'm sorry. What does that mean?

**Pull request closed without merging**